aspnetcore/signalr/supported-platforms.md

```diff
@@ -0,0 +1,37 @@
+---
+title: ASP.NET Core SignalR supported platforms
+author: rachelappel
+description: Supported platforms for ASP.NET Core SignalR 
+manager: wpickett
```

Comment:
Add the following metadata under the `manager` metadata:

`monikerRange: '>= aspnetcore-2.1'`

This causes the doc to display only when 2.1 (or later) is selected from the version selector.